modules/installation-configuration-parameters.adoc

```diff
@@ -1586,6 +1590,48 @@ In {product-title} 4.12 and later, the `apiVIP` configuration setting is depreca
 In {product-title} 4.12 and later, the `ingressVIP` configuration setting is deprecated. Instead, use a list format to enter a value in the `ingressVIPs` configuration setting.
 ====
 
+ifdef::vsphere-zones[] 
+|`platform.vsphere.failureDomains`
+|Optional. The list of failure domain definitions.
+|Array of `failureDomain` definitions.
+
+|`platform.vsphere.failureDomains.*.name`
+|Required. The name of the failure domain. This name will be referred to by the machine pools.
+|String
+
+|`platform.vsphere.failureDomains.*.region`
+|Required. The value of the openshift-region tag assigned to the topology for the failure failure domain.
+|String
+
+|`platform.vsphere.failureDomains.*.zone`
+|Required. The value of the openshift-zone tag assigned to the topology for the failure failure domain.
+|String
+
+|`platform.vsphere.failureDomains.*.topology.datacenter`
+|Required. The datacenter associated with the failure domain. If not defined, the datacenter defaults to `platform.vsphere.datacenter`.
+|String
+
+|`platform.vsphere.failureDomains.*.topology.computeCluster`
+|Required. The compute cluster associated with the failure domain. If not defined, the compute cluster is derived from `platform.vsphere.cluster` and `platform.vsphere.datacenter`.
```

Comment:
```suggestion
|Required. The compute cluster associated with the failure domain. If not defined, the compute cluster takes the value of `platform.vsphere.cluster` and `platform.vsphere.datacenter`.
```
Reworded to avoid passive voice. See if you can apply this change to the other descriptions below that follow the same format.